### PR TITLE
chore: improve display of lock screen, closes #4606

### LINKED ISSUE #4606: Fix password-related formatting issues
Related to fixes made with https://github.com/leather-wallet/extension/issues/4423

Across the unlock and set-password screens respectively.

## Spacing issues

Screenshot 2023-11-30 at 12 44 55

## Alignment issue

Screenshot 2023-11-30 at 12 44 32

## Alignment  issue

Screenshot 2023-11-30 at 12 44 28

### PR DESCRIPTION
**Pull request by pete-watters**
> Try out this version of Leather — [Extension build](https://github.com/leather-wallet/extension/actions/runs/7260526458), [Test report](https://leather-wallet.github.io/playwright-reports/fix/4606/password-alignment)

This PR adds some improvements to the UI of:
- lock screen
- password strength indicator

https://github.com/leather-wallet/extension/assets/2938440/f973e42f-951e-46bf-a42c-14b62b8133d4

Screenshot 2023-12-18 at 12 31 15

### CODE REVIEW

File: src/app/components/secret-key/two-column.layout.tsx [hunk 1]
```diff
@@ -26,7 +26,9 @@ export function TwoColumnLayout({
         gap="space.07"
         mx={['auto', 'space.03', 'space.03', 'space.03']}
       >
-        <styled.div maxWidth="440px">{leftColumn}</styled.div>
+        <styled.div minWidth="440px" maxWidth="440px" textAlign="left">
```
Comment (fbwoolf) [inline src/app/components/secret-key/two-column.layout.tsx]: Would setting the min and max width here as the same be just a fixed width of `440px`? 🤔

**Review by fbwoolf — APPROVED** [review]

**Review by kyranjamie — APPROVED** [review]
I would suggest another improvement would be not having the button jump position when a validation error is shown

File: src/app/components/secret-key/two-column.layout.tsx (hunk 1 above)
Comment (pete-watters) [reply src/app/components/secret-key/two-column.layout.tsx]: Thanks @fbwoolf 🤦 the worst part is I saw this but thought I was avoiding a set width. I'll update it

**Comment by kyranjamie** [comment]
Mmm, forgot some of the lock screen issues will've been fixed here

**Comment by pete-watters** [comment]
> Mmm, forgot some of the lock screen issues will've been fixed here

Did you want to release this? I can do another release. This is just some minor visual tweaks

**Comment by kyranjamie** [comment]
It's been released already so why not!

**Pull request merged**